Dockerfile.plugins

```diff
@@ -1,32 +1,33 @@
 FROM golang:1.12
 ENV GO111MODULE on
 
-# Kinesis Streams
 ARG KINESIS_PLUGIN_CLONE_URL=https://github.com/aws/amazon-kinesis-streams-for-fluent-bit.git
-ARG KINESIS_PLUGIN_BRANCH=mainline
+ARG KINESIS_PLUGIN_TAG=v1.5.1
+ARG FIREHOSE_PLUGIN_CLONE_URL=https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit.git
+ARG FIREHOSE_PLUGIN_TAG=v1.4.1
+ARG CLOUDWATCH_PLUGIN_CLONE_URL=https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit.git
+ARG CLOUDWATCH_PLUGIN_TAG=v1.3.2
```

Comment:
I prefer them (tag lines) moving to a different place; or read from the plugin repos.

Reply:
Otherwise, we can easily miss or mess up.

Reply:
Could you elaborate? I don't understand what you meant.

Reply:
Rather than putting hard coded plugin versions, we can read the latest release tag from original plugin repositories.

Reply:
If we want to go with this, please add a section in release process doc to update these tags.